nkms/data_sources.py

```diff
@@ -0,0 +1,15 @@
+from nkms.crypto.api import encrypt_and_sign
+
+
+class DataSource:
+
+    def __init__(self, policy_pubkey_enc, signer):
+        self._policy_pubkey_enc = policy_pubkey_enc
```

Comment:
Is `policy_pubkey_enc` the `UmbralPublicKey` of Alice? If that's the case, I think the name is not very good, since the same `UmbralPublicKey` can be used in more than one policies.

Reply:
Hmm... does anything about `policy_pubkey_enc` make it sound like it is exclusive to this `Policy`?  It's just the public key of this (and perhaps other) policies.